lib/ivis_opengl/gfx_api_gl.cpp

```diff
@@ -0,0 +1,87 @@
+#include "gfx_api.h"
+
+#include <GL/glew.h>
+#include <algorithm>
+#include <cmath>
+
+namespace gl_api

```

Comment:
Not a big fan of using namespaces in c++ files like this. Both because it clashes with existing style, causing unnecessary style changes when touching code, and because it leads to a lot of unnecessary indentation in general. Please use the <namespace>::<function name> declaration style instead (as you already do in the last function).